receiver/prometheusreceiver/config_test.go

```diff
@@ -54,6 +54,7 @@ func TestLoadConfig(t *testing.T) {
 	assert.Equal(t, r1.PrometheusConfig.ScrapeConfigs[0].JobName, "demo")
 	assert.Equal(t, time.Duration(r1.PrometheusConfig.ScrapeConfigs[0].ScrapeInterval), 5*time.Second)
 	assert.Equal(t, r1.UseStartTimeMetric, true)
+	assert.Equal(t, r1.StartTimeMetricRegex, ".*_process_start_time_seconds$")
```

Comment:
```suggestion
	assert.Equal(t, r1.StartTimeMetricRegex, ".*process_start_time_seconds$")
```
The leading undescore `_process...` means that the standard name `process_start_time_seconds` won't match it.
This doesn't reallly matter for testing, but would have been important in the actual use.

Reply:
Update the pattern to reflect the prefixes must end with `_` in case some metrics are called things like `subprocess_start_time_seconds` for whatever reason :)